contributors/devel/development.md

```diff
@@ -101,35 +119,56 @@ revising your hypothesis.  The above issues do this transparently, using figures
 and data rather then conjecture. Notice that the problem is analyzed and a
 correct solution is created before a single line of code is reviewed.
 
-## Building Kubernetes with Docker
+## Getting the code
+
+Kubernetes is developed using git for version control.
+
+### GitHub workflow
+
+To check out code to work on, please refer to [this guide](/contributors/guide/github-workflow.md).
+
+For tips, tricks, and common practices used within the Kubernetes project read the [Kubernetes GitHub Cheat Sheet][github-cheat-sheet].
+
+## Building Kubernetes
+
+There are two recommended ways of building Kubernetes, and the choice of one or the other depends on if you do plan on developing upstream. One way is building a release from source, this is recommended for development purposes. If there is no development involved, it is suggested to use a pre-built version of the current release.
+
+### Building Kubernetes with Docker
 
 Official releases are built using Docker containers. To build Kubernetes using
 Docker please follow [these
 instructions](http://releases.k8s.io/HEAD/build/README.md).
 
-## Building Kubernetes on a local OS/shell environment
+### Building Kubernetes on a local OS/shell environment
 
 Kubernetes development helper scripts assume an up-to-date GNU tools
-environment. Recent Linux distros should work out-of-the-box.
+environment. Recent Linux distributions should work out-of-the-box.
```

Comment:
are the dev helper scripts the same ones that live in http://git.k8s.io/kubernetes/build/README.md, according to Docker instructions?

Or is this a separate set of scripts? My guess is no; but I don't know. If there _are_ separate scripts, it would be good to link to them.